.github/workflows/lint-build-test.yml

```diff
@@ -30,19 +30,14 @@ jobs:
           path: ${{ steps.yarn-cache-dir.outputs.YARN_CACHE_DIR }}
           key: yarn-cache-${{ runner.os }}-${{ steps.yarn-version.outputs.YARN_VERSION }}-${{ hashFiles('yarn.lock') }}
       - run: yarn --immutable
-      - run: yarn build
```

Comment:
I noticed that you moved the lint ahead of the build in the create-release-branch repo as well. Was this just so that it would fail more quickly for lint errors?

Reply:
Yes, exactly. I can move this to a new PR since it's not critical for this PR.

Reply:
Okay, original file restored here: [d2c700e](https://github.com/MetaMask/controllers/pull/831/commits/d2c700eb95d1b696cb7632650cae9ae1f9b59727). And PR created here: https://github.com/MetaMask/controllers/pull/939